Remove a message from stderr

jan-cerny opened the pull request:
When a remediation can't find the respective rule YML, it produces
a message on stderr, which pollutes the build output. In fact, this
is a normal situation when the remediation exists but the rule
isn't a part of the given product.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/a80d8f60-7c84-438c-894b-7d8528e3d50e) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/a80d8f60-7c84-438c-894b-7d8528e3d50e) completed: **1 new issues**

jan-cerny commented:
@openscap-ci test this please

matusmarhefka commented:
@jan-cerny Can we rather disable the error only in case that the rule isn't a part of the given product and leave the error in other cases?

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/47673787-7f3f-4acc-a87a-ab473b570695) was created.

jan-cerny commented:
@matusmarhefka I have changed it to check the existence in the caller instead. If user uses the method in its own code with a nonexistent path it will hopefully raise him an exception. I also removed the unused code that I noticed.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/47673787-7f3f-4acc-a87a-ab473b570695) completed: **3 new issues**

jan-cerny commented:
@openscap-ci test this please

matusmarhefka commented on ssg/build_remediations.py:
```diff
@@ -258,10 +246,6 @@ class BashRemediation(Remediation):
-        pass
-
 
 class AnsibleRemediation(Remediation):
```
Add `isfile` check also into the `from_snippet_and_rule` method of this class.

jan-cerny replied:
OK, I will do that.

jan-cerny replied:
done

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/c9a9e891-7d32-445f-a130-a907d59d1c78) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/c9a9e891-7d32-445f-a130-a907d59d1c78) completed: **2 new issues**

matusmarhefka commented:
@jan-cerny LGTM, thanks.
